index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+</head>
```

Comment:
kindly you should have a title before the end of your header section according to the project description. please be aware that the title in the header makes the user see the title of the page in the browser. it is very important to familiarize yourself with this practice.